.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules/
+dist/
```

Comment:
Do you plan to only rollup files and publish via Travis, you don't want to put these on Git too? I see SDK UI does it: https://github.com/onfido/onfido-sdk-ui/tree/master/dist

Reply:
I don't think there's any reason to commit dist files anymore? Either install via npm or get from e.g. unpkg